HST/run_coverage.sh

```diff
@@ -0,0 +1,2 @@
+coverage run -m pytest query_mast/tests
```

Comment:
You should have a .coveragerc that sets up parameters and also turns on branch analysis. See OPUS or pds-webtools for examples.